tests/test_pipeline_media.py

```diff
@@ -176,257 +143,8 @@ def test_should_remove_req_res_references_before_caching_the_results(self):
         self.assertIsNone(context)
 
 
-class MockedMediaPipeline(MediaPipeline):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self._mockcalled = []
-
-    def download(self, request, info):
-        self._mockcalled.append("download")
-        return super().download(request, info)
-
-    def media_to_download(self, request, info, *, item=None):
-        self._mockcalled.append("media_to_download")
-        if "result" in request.meta:
-            return request.meta.get("result")
-        return super().media_to_download(request, info)
-
-    def get_media_requests(self, item, info):
-        self._mockcalled.append("get_media_requests")
-        return item.get("requests")
-
-    def media_downloaded(self, response, request, info, *, item=None):
-        self._mockcalled.append("media_downloaded")
-        return super().media_downloaded(response, request, info)
-
-    def media_failed(self, failure, request, info):
-        self._mockcalled.append("media_failed")
-        return super().media_failed(failure, request, info)
-
-    def item_completed(self, results, item, info):
-        self._mockcalled.append("item_completed")
-        item = super().item_completed(results, item, info)
-        item["results"] = results
-        return item
-
-
-class MediaPipelineTestCase(BaseMediaPipelineTestCase):
```

Comment:
On the first glance this, or maybe parts of this, looks like tests for the `MediaPipeline` machinery so they need to stay (unless the same tests are also in some other class/file).

Reply:
@wRAR I’ve kept required tests. Let me know if im missing something